### PR TITLE
Update UI:

### PR DESCRIPTION
**Pull request by sofyakurilova**
linters+prettier added; unused files removed

### CODE REVIEW

**Review by jboddey — CHANGES_REQUESTED** [review]

File: modules/ui/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -13,37 +13,39 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import {NgModule} from '@angular/core';
-import {RouterModule, Routes} from '@angular/router';
-import {allowToRunTestGuard} from './guards/allow-to-run-test.guard';
-import { HashLocationStrategy, LocationStrategy } from '@angular/common';
```
[the remark this answers is not in the document]
Comment (sofyakurilova) [reply modules/ui/src/app/app-routing.module.ts:19]: { useHash: true } do the same thing on line 48

File: modules/ui/angular.json [hunk 2]
```diff
@@ -17,21 +17,14 @@
         "build": {
           "builder": "@angular-devkit/build-angular:browser",
           "options": {
-            "outputPath": "dist",
+            "outputPath": "dist/test-run-ui",
```
[the remark this answers is not in the document]
Comment (sofyakurilova) [reply modules/ui/angular.json:20]: Sure

**Review by jboddey — APPROVED** [review]

**Comment by jboddey** [comment]
Just awaiting CLA. Thanks @sofyakurilova

**Comment by jboddey** [comment]
Closing this PR for now. The UI will be updated in another PR.

**Pull request closed without merging**